Libplanet/Blockchain/BlockChain.cs

```diff
@@ -1396,15 +1393,15 @@ internal BlockLocator GetBlockLocator(int threshold = 10)
             );
 
             bool reorged = !Tip.Equals(topmostCommon);
-            if (renderBlocks && reorged)
+            if (render && reorged)
```

Comment:
Is it okay we unify `renderBlocks` and `renderActions` to just `render`?

Reply:
I also don't have any idea yet. (just check 😅 )

Reply:
Looks no problem to me.